google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -42,12 +42,16 @@
   static final String COUNT = "1";
 
   // The Metric name and description
-  static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_session";
+  static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_sessions";
   static final String MAX_ALLOWED_SESSIONS = "cloud.google.com/java/spanner/max_allowed_sessions";
   static final String IN_USE_SESSIONS = "cloud.google.com/java/spanner/in_use_sessions";
+  static final String GET_SESSION_TIMEOUTS = "cloud.google.com/java/spanner/get_sessions_timeouts";
```

Comment:
There's a discrepancy in the naming here.

```suggestion
  static final String GET_SESSIONS_TIMEOUTS = "cloud.google.com/java/spanner/get_sessions_timeouts";
```

Reply:
I would change get_sessions_timeouts to get_session_timeouts instead.

Reply:
I don't have strong preference, but **sessions** seems consistent across other metrics. @skuruppu thoughts?

Reply:
I think it should be `get_session_timeouts`.

Reply:
done, ptal